airbyte-cdk/python/airbyte_cdk/sources/concurrent_source/concurrent_read_processor.py

```diff
@@ -144,14 +164,6 @@ def start_next_partition_generator(self) -> Optional[AirbyteMessage]:
             stream = self._stream_instances_to_start_partition_generation.pop(0)
             self._thread_pool_manager.submit(self._partition_enqueuer.generate_partitions, stream)
             self._streams_currently_generating_partitions.append(stream.name)
-            self._logger.info(f"Marking stream {stream.name} as STARTED")
-            self._logger.info(f"Syncing stream: {stream.name} ")
-            return stream_status_as_airbyte_message(
```

Comment:
the message is now emitted when receiving a stream availability object, not when submitting the partition generation task